version: ~> 1.0
import: collective/buildout.plonetest:travis/default.yml
# TravisCI fails with this message:
# The command "sh -e /etc/init.d/xvfb start" failed and exited with 127 during .
# before_script:
#   - export DISPLAY=:99.0
#   - sh -e /etc/init.d/xvfb start
matrix:
  include:
    - python: "2.7"
      env: PLONE_VERSION="4.3"
    - python: "2.7"
      env: PLONE_VERSION="5.2"
    - python: "3.8"
      env: PLONE_VERSION="5.2"
